hail/src/main/scala/is/hail/expr/ir/CompileAndEvaluate.scala

```diff
@@ -19,6 +19,18 @@ object CompileAndEvaluate {
     }
   }
 
+  def applyWithType[T](ctx: ExecuteContext,
```

Comment:
would prefer to inline the usage in TableParallelize, since this is only used in one place and it'll get removed when we're done lowering.

Reply:
kk